static/app/utils/profiling/flamegraph/flamegraphStateProvider/index.tsx

```diff
@@ -280,3 +329,36 @@ export function FlamegraphStateProvider(
     </FlamegraphStateContext.Provider>
   );
 }
+
+type Candidate = {
+  score: number;
+  threadId: number | null;
+};
+
+function scoreFlamegraph(
+  flamegraph: Flamegraph,
+  focusFrame: FlamegraphProfiles['focusFrame']
+): number {
+  if (!defined(focusFrame)) {
+    return 0;
+  }
+
+  let score = 0;
+
+  const frames: FlamegraphFrame[] = [...flamegraph.root.children];
+  while (frames.length > 0) {
+    const frame = frames.pop()!;
+    if (
+      frame.frame.name === focusFrame.name &&
+      frame.frame.image === focusFrame.package
```

Comment:
Is it expected to compare `image` and `package`?

Reply:
yes, we're using more platform agnostic names when we introduced the new functions table